jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java

```diff
@@ -57,6 +57,7 @@ public static String getDriverFolder() {
     public static String BROWSER_SIZE = "MAXIMIZE";
     public static final String DEFAULT_DRIVER = "chrome";
     public static String DRIVER_NAME = DEFAULT_DRIVER;
+    public static String ARGUMENTS_PROPERTY = "arguments";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4714033)